blog/2024-03-04-on-implementation-of-distributed-prtocols/index.mdx

```diff
@@ -5,6 +5,8 @@ authors: sergefdrv
 tags: ["technical", "overview", "long"]
 toc_max_heading_level: 4
 image: "./image.png"
+draft: false
+comments: true # for Disqus
```

Comment:
No need to explicitly set it: `comments` is `true` by default in `src/theme/BlogPostItem/index.js`.